bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -138,6 +138,7 @@ public class Display extends Device implements Executor {
 	EventTable eventTable, filterTable;
 	boolean useOwnDC;
 	boolean externalEventLoop; // events are dispatched outside SWT, e.g. TrackPopupMenu or DoDragDrop
+	boolean rescalingAtRuntime;
```

Comment:
as this field is exposed via API, we should make it private, shouldn't we? Especially as set setter contains important logic

Reply:
Totally makes sense. I just forgot about that because all other fields in the class are package protected, but making it private is totally correct. Fixed in 784ffc7.